### PR TITLE
Nirvana jvm

### PR DESCRIPTION
**Pull request by AsterNighT**
Signed-off-by: AsterNighT <klxjt99@outlook.com>

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code
```bash
kubectl create namespace app
kubectl label ns app admission-webhook=enabled
kubectl apply -f  sidecar.yaml
kubectl apply -f  sidecar-template.yaml
kubectl apply -f  app.yaml
<wait until the app is ready>
kubectl -n app port-forward pod/springboot-jvmchaos-demo-<some-id> 8080:8080&
curl localhost:8080/hello #yields "Hello friend"
kubectl apply -f  jvm-return-example.yaml
curl localhost:8080/hello #yields "Hello chaos mesh!"
kubectl delete -f  jvm-return-example.yaml
curl localhost:8080/hello #yields "Hello friend"
```
Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by AsterNighT** [comment]
And I wonder how to test this.

**Comment by AsterNighT** [comment]
I manually tested it and it seems to work well. There are so many preparations to do and I wonder how can we code this into e2e.

**Comment by YangKeao** [comment]
> I manually tested it and it seems to work well. There are so many preparations to do and I wonder how can we code this into e2e.

Are you sure that you have tested? This example cannot pass the validation (`scheduler` is an unknown field). Have you checked the image ID?

**Comment by YangKeao** [comment]
> > I manually tested it and it seems to work well. There are so many preparations to do and I wonder how can we code this into e2e.
> 
> Are you sure that you have tested? This example cannot pass the validation (`scheduler` is an unknown field). Have you checked the image ID?

Fixed now. LGTM :+1:

File: examples/jvm/jvm-return-example.yaml [hunk 1]
```diff
@@ -16,5 +16,3 @@ spec:
     labelSelectors:
       app: springboot-jvmchaos-demo
   duration: "50s"
-  scheduler:
-    cron: "@every 1m"
```
Comment (Gallardot) [inline examples/jvm/jvm-return-example.yaml:20]: Why delete `cron` in the example?

Comment (AsterNighT) [reply examples/jvm/jvm-return-example.yaml:20]: Sry I forgot to upload the yaml

**Comment by AsterNighT** [comment]
> > I manually tested it and it seems to work well. There are so many preparations to do and I wonder how can we code this into e2e.
> 
> Are you sure that you have tested? This example cannot pass the validation (`scheduler` is an unknown field). Have you checked the image ID?

whoops, I forgot to update the example.

File: examples/jvm/jvm-return-example.yaml (hunk 1 above)
Comment (AsterNighT) [reply examples/jvm/jvm-return-example.yaml:20]: `cron` part is delegated to `schedule` in nirvana. I feel it unnecessary to test it here.

Comment (Gallardot) [reply examples/jvm/jvm-return-example.yaml:20]: got it.maybe we should update the other examples.

Comment (YangKeao) [reply examples/jvm/jvm-return-example.yaml:20]: An issue #1887 has been added.

File: cmd/chaos-builder/schedule.go [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
```
[the remark this answers is not in the document]
Comment (YangKeao) [reply cmd/chaos-builder/schedule.go:19]: ignore

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**